### DISCUSSION TITLE
Causes chkdsk to report false positives in Win 11 24H2

### DISCUSSION THREAD

**Discussion by leftstack** [openzfsonwindows/openzfs#427] [Testing]
Upon uninstalling OpenZFS (rc11), all chkdsk errors (false positives) immediately cease and chkdsk once again reports 0 errors on the system volume. As of now, this does not seem to affect volumes /other/ than the system volume (typically c:).

**Comment by lundman** [comment]
This is likely related to VSS issue, in that you can't really run chkdsk at all. https://github.com/openzfsonwindows/openzfs/issues/406

**Comment by leftstack** [comment]
Thanks for that. I'll keep disabling VSS in mind during further testing.
